images/runtime/Dockerfile

```diff
@@ -35,7 +35,8 @@ ENV FORCE_BUILD=1
 
 # Update ubuntu packages to the most recent versions
 RUN apt-get update && \
-    apt-get upgrade -y
+    apt-get upgrade -y && \
```

Comment:
This changes requires a couple of more steps as they are explained [here](https://docs.cilium.io/en/v1.11/contributing/development/images/#update-cilium-builder-and-cilium-runtime-images). Please follow them so that we have all image digests up-to-date. Thanks!

Reply:
Apologies for missing these steps. I did the first part of it - unfortunately I can't see the members of team build, so I don't know who to ping.

Reply:
You can simply post a message in #development on Slack in such cases and someone will help :slightly_smiling_face: